src/hawkbit-client.c

```diff
@@ -867,3 +868,4 @@ int hawkbit_start_service_sync()
 
         return res;
 }
+
```

Comment:
Please remove the added extra line at the end of the file. Everything else looks good to me.